slick-testkit/src/codegen/resources/dbs/uuid-h2.sql

```diff
@@ -1,3 +1,5 @@
 create table "person" ("id" INTEGER NOT NULL PRIMARY KEY,
```

Comment:
@A1kmm copied uuid.sql to uuid-h2.sql and uuid-postgres.sql and those are also the only two backends where this test file was used, so the rename seems fine with me.